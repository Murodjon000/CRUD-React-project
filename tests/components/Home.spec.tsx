import shallowSetup from "./mock.spec";
import toJson from "enzyme-to-json";

test("should render Header component", () => {
  const { homeWrapper } = shallowSetup();

  expect(homeWrapper.find("div").first().hasClass("test-home-wrapper")).toBe(
    true
  );
});

test("should render Home component Snapshot", () => {
  const { homeWrapper } = shallowSetup();

  expect(toJson(homeWrapper)).toMatchSnapshot();
});
