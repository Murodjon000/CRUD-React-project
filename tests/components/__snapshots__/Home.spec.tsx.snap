// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Details component Snapshot 1`] = `
<div
  className="project__details-wrapper"
>
  <div
    className="project__details-wrapper-image"
  >
    <img
      alt=""
      className="test-img"
      src="https://miro.medium.com/max/1000/1*9xsPgsjcFOnwhgPwtsO_HQ.jpeg"
    />
  </div>
  <div
    className="project__details-wrapper-box"
  >
    <h1
      className="test-title"
    >
      Where can I get some?
    </h1>
    <p
      className="test-desc"
    >
       
      ðŸ‘€ PageMaker including versions of Lorem Ipsum.
    </p>
    <button
      className="test-btn"
      onClick={[Function]}
      type="button"
    >
      Back to home
    </button>
  </div>
</div>
`;

exports[`should render Home component Snapshot 1`] = `
<div
  className="test-home-wrapper"
>
  <Connect(ProjectsForm) />
  <Connect(ProjectsList) />
</div>
`;
